### ISSUE TITLE
Test Case contribution and Clearing the Question statements for 902. Numbers At Most N Given Digit Set 

### ISSUE THREAD

**Issue by dheeraj2645** [LeetCode-Feedback/LeetCode-Feedback#1416]
#### Your LeetCode username
Heisen_berg1

#### Category of the bug
- [- ] Question
- [- ] Solution
- [ ] Language

#### Description of the bug
A test case is showing wrong answer from expected answer for my code.Which is succesfully got submitted.This indicates this test case is being missed in your test cases.So i want you to include this test case.

#### Code you used for Submit/Run operation

```
class Solution {
public:
    int atMostNGivenDigitSet(vector<string>& digits, int n) {
        sort(digits.begin(), digits.end());
        string numstr = to_string(n);
        int count = 0;
        
        for(int i = 1; i <= numstr.size() - 1; i++) {
   
            count += pow(digits.size(), i);
        }
        
        int i = 0;
        while(i < numstr.size()) {
            int j = 0;
            while(j < digits.size() && digits[j][0] < numstr[i]) {
                count += pow(digits.size(), numstr.size() - 1 - i);
                
                j++;
            }
            if(j == digits.size() || digits[j][0] > numstr[i]) {
                break;
            }
            i++;
        }

        if(i == numstr.size()) {
            count++;
           
        }
        return count;
    }
};
```

#### Language used for code
C++ 

#### Expected behavior
The expected behavior for the following test case is different from the original answer.
Input : ["1","9","4"] , 42
Actual answer : 7
Expected answer showing : 10

#### Screenshots
![image](https://user-images.githubusercontent.com/66384202/99875684-77295700-2c17-11eb-9de1-bff659ffc96c.png)
![image](https://user-images.githubusercontent.com/66384202/99875708-ac35a980-2c17-11eb-99d8-d5e831af774c.png)

#### Additional context
So, Basically what I see here is In all the test cases provided for this question the array of the strings is considered as sorted array. But it is not mentioned in the question. That is why many manual test cases are giving wrong expected answers. So either you can update your solution code or you can specify in the question that input array is sorted.

**Comment by lc-kay** [comment]
Hi @dheeraj2645 
Thank you for reaching out to us. I've relayed this issue to our team to investigate.

**Comment by lc-kay** [comment]
Hi @dheeraj2645 
Thank you for your time. The team reviewed your report and noticed the problem was missing constraints and validations. After adding them, the test case you contributed becomes invalid so we didn't add the test case. We appreciate your support!
